internal/controller/pdb_to_evictionautoscaler_controller.go

```diff
@@ -59,6 +60,23 @@ func (r *PDBToEvictionAutoScalerReconciler) Reconcile(ctx context.Context, req r
 	// Track PDB existence
 	metrics.PDBCounter.WithLabelValues(pdb.Namespace, createdByUsStr).Inc()
 
+	// Check if eviction autoscaler should be enabled for this PDB
+	// Enable by default in kube-system namespace, otherwise check annotation on the namespace
+	if pdb.Namespace != KubeSystemNamespace {
+		// Fetch the namespace to check for the annotation
+		namespace := &corev1.Namespace{}
+		err = r.Get(ctx, k8s_types.NamespacedName{Name: pdb.Namespace}, namespace)
```

Comment:
share this code with between recocnilrs with a 

func isEnabled(client , namespace string , whitelistednsamepsace []string) (error, bool)